### DISCUSSION TITLE
Why have analytics not been ported yet?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [GitLiveApp/firebase-kotlin-sdk#203] [General]
Okay to clarify the title because it sounds kinda rude and assertive, I am genuinely wondering why have not the analytics been ported yet? Is there anything different to analytics that prevents the usual approach from the other services? Are they just not popular enough to be worth the hassle?

**Comment by nbransby** [comment]
Haha np, yes its because not popular enough, implementation would not differ from the others, PR are welcome!
